schema/objects/drilling-campaign/1.0.0/drilling-campaign.schema.json

```diff
@@ -0,0 +1,187 @@
+{
+  "$id": "/objects/drilling-campaign/1.0.0/drilling-campaign.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "A planned drillholes and interim drillhole data for a drilling campaign.",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "/components/base-spatial-data-properties/1.1.0/base-spatial-data-properties.schema.json"
+    },
+    {
+      "properties": {
+        "schema": {
+          "const": "/objects/drilling-campaign/1.0.0/drilling-campaign.schema.json"
+        },
+        "depth_units": {
+          "description": "The units of depth for the drillholes.",
+          "$ref": "/elements/unit/1.0.1/categories/unit-length.schema.json"
+        },
+        "hole_id": {
+          "description": "Hole IDs.",
+          "$ref": "/components/category-data/1.0.1/category-data.schema.json"
+        },
+        "planned": {
+          "description": "Planned data for the drilling campaign.",
+          "allOf": [
+            {
+              "properties": {
+                "collar": {
+                  "description": "Collar data for the planned drillholes.",
+                  "$ref": "#/$defs/collar"
+                },
+                "path": {
+                  "description": "The path taken by the downhole segment. Columns: distance, inclination, azimuth, dip, toolface, lift rate, drift rate.",
+                  "allOf": [
+                    {
+                      "$ref": "/elements/float-array-md/1.0.1/float-array-md.schema.json",
+                      "properties": {
+                        "width": {
+                          "description": "number of columns",
+                          "const": 7
+                        }
+                      },
+                      "required": [
+                        "width"
+                      ]
+                    },
+                    {
+                      "$ref": "/components/attribute-list-property/1.2.0/attribute-list-property.schema.json"
+                    }
+                  ]
+                },
+                "results": {
+                  "description": "Planned attributes for the drilling campaign.",
+                  "type": "array",
+                  "oneOf": [
```

Comment:
I think possibly this should be stored within an `items` block, as in the Downhole Collection:
https://github.com/SeequentEvo/evo-schemas/blob/main/schema/objects/downhole-collection/1.3.0/downhole-collection.schema.json#L82

@ArthurGW can you check me on that?

Thanks!